server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/DefaultOrganizationResourceMigrator.java

```diff
@@ -100,21 +98,16 @@ public void migrate(@NonNull User user) {
         }));
         this.transactionTemplate.execute((transactionStatus -> {
             try {
-                Organization team = organizationService
-                        .getByOrganizationTypeAndUserId(OrganizationType.TEAM, user.getId())
-                        .orElseThrow(() -> new RuntimeException("User doesn't belong to any TEAM organization"));
-                Organization individual = new Organization();
-                individual.setId(-1L);
-                individual.setType(OrganizationType.INDIVIDUAL);
-                boolean hasIndividualPermission =
-                        verticalPermissionValidator.implies(individual, Arrays.asList("read", "update"), team.getId());
-                Optional<Organization> organization = organizationService.getByOrganizationTypeAndUserId(
-                        OrganizationType.INDIVIDUAL, user.getId());
-                if (!organization.isPresent() && hasIndividualPermission) {
-                    log.info("Individual organization not found, start to initialize for userId={}", user.getId());
-                    OrganizationEntity saved = individualOrganizationMigrator.migrate(user);
-                    log.info("Initialized individual organization successfully for userId={}", user.getId());
-                }
+                organizationService.getByOrganizationTypeAndUserId(
```

Comment:
the return value has never acquired, we may do as follow:
```Java
Optional< Organization> optional = organizationService.getByOrganizationTypeAndUserId(OrganizationType.INDIVIDUAL, user.getId());
if (!optional.isPresent()) {
     log.info("Individual organization not found, start to initialize for userId={}",user.getId());
     OrganizationEntity saved = individualOrganizationMigrator.migrate(user);
     log.info("Initialized individual organization successfully for userId={}, organizationId={}", user.getId(), saved.getId());
}
```

this way may be much more clearer.

by the way, 'verticalPermissionValidator' is never acquired, we'd better delete it.

Reply:
okay, done